lib/run_cucumber.js

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const log = require('./utils/log');
+const session_vars = require('./utils/session_vars');
+
+// import {loadConfiguration, loadSupport, runCucumber } from '@cucumber/cucumber/api';
+const cucumber_api = require('@cucumber/cucumber/api');
+
+(async function main() {
+  let argv = process.argv;
+
+  let just_args = argv.slice(2);
+
+  const environment = { cwd: process.cwd() };
+  const provided_config = {
+    "publishQuiet": true,
+    "require": "./lib/index.js", 
+    "format": ["progress", "summary:cucumber-report.txt"],
+    "paths": ["docassemble/*/data/sources/*.feature"],
+    "retry": 1,
+  }
+  if (just_args.length > 0) {
+    // Cucumber expects tags in boolean expressions, like (@a0 or @a1) and @random.
+    // If we get a list of only tags, then "or" them all together
+    if (just_args.every((val) => val.startsWith("@"))) {
+      provided_config.tags = just_args.join(' or ');
+    }
+    // If we get a list of seemingly no tags, assume they meant to use tags
+    else if (just_args.every((val) => !val.startsWith("@"))) {
+      provided_config.tags = just_args.map((val) => "@" + val).join(' or ');
+    }
+    // If it's a mix of tags and no tags, it's probably a boolean expression!
+    else {
+      provided_config.tags = just_args.join(' ');
+    }
+  }
```

Comment:
Can we just pass the tags through as they are? Not sure we should deviate from the documented behavior of tags/try to manipulate after the fact.

Reply:
We can't pass tags as they are, as it wouldn't run as is. This tries to keep the behavior as or-ing things, which is what the cli currently does.

Reply:
Tested this, and these all work through GitHub actions as well